lib/linguist/heuristics.yml

```diff
@@ -216,6 +216,13 @@ disambiguations:
     pattern: '^<\?(?:php)?'
   - language: SourcePawn
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
+  - language: NASL
+    pattern:
+    - '^\s*include\s*\(\s*(?:"|'')[\\/\w\-\.:\s]+(?:"|'')\s*\)\s*;'
+    - '^\s*(?:global|local)_var\s+(?:[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)(?:,\s*[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)*+\s*;'
+    - '^\s*namespace\s+[\w\-]+\s*(?:$^)*\s*\{'
+    - '^\s*object\s+[\w\-]+\s*(?:extends\s+[\w\-:]+\s+|)\s*(?:$^)*\s*\{'
```

Comment:
Another empty expression. See my first comment.

(Note that `{` doesn't need to be escaped unless it's followed by a numbers and a closing brace: `{1}`, `{1,2}`, `{3,}`, etc).

Reply:
Changed in latest commit.